welcome/welcome.py

```diff
@@ -412,7 +412,7 @@ def check(message: discord.Message):
             await ctx.send(f"{name} removed from list")
             await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         else:
-            ctx.send(f"{name} not in list of greetings")
+            await ctx.send(f"{name} not in list of greetings")
```

Comment:
Can you put this in a separate PR and merge this into master, since this is not directly related to dpy2?